crypto/bls12381_utils.go

```diff
@@ -222,3 +217,28 @@ func checkInG1Test(pt *pointG1) bool {
 func benchG1Test() {
 	_ = C.subgroup_check_G1_bench()
 }
+
+// This is only a TEST function.
+// It hashes `data` to a G1 point using the tag `dst` and returns the G1 point serialization.
+// The function uses xmd with SHA256 in the hash-to-field.
```

Comment:
Same comment as the above: `blst` can encode-to-map from uniform field bytes, so that we can test kmac + relic map-to-curve + relic sign against kmac + bls map-to-curve and sign.

Reply:
This function is only there to test our flow map-to-curve (which is now relic's) against the test vector in the draft. The test vector uses xmd_sha256 so if we want to get the same results we would need to call xmd_sha256. 

This function is not used in any test related to testing signatures or to compare to BLST.

Reply:
oh, yes of course! that makes sense, my bad.